lib/run.js

```diff
@@ -14,7 +14,7 @@ const probot = createProbot({
   id: process.env.APP_ID,
   secret: process.env.WEBHOOK_SECRET,
   cert: findPrivateKey(),
-  port: process.env.PORT || 3000,
+  port: process.env.UGH_PORT || process.env.PORT || 3000,
```

Comment:
Overmind overrides `PORT` with [some other value](https://github.com/DarthSim/overmind#specifying-the-ports). It's a bit annoying in this scenario. I've added this as a temporary workaround.

Very open to suggestions!